tekton/cronjobs/robocat-triggers-deploy-latest-cron/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+bases:
+- ../tekton-service-cd-cron-base
+patchesStrategicMerge:
+- cronjob.yaml
+nameSuffix: "-robotcat-triggers"
```

Comment:
hmm this looks very similar to https://github.com/tektoncd/plumbing/blob/master/tekton/cronjobs/robocat-pipeline-deploy-latest-cron/kustomization.yaml

Do you think the cronjobs that use the same bases `- ../tekton-service-cd-cron-base` could live in one folder? I'm wondering if we could skip a layer of kustomization.yamls files

Reply:
Each overlay needs its own folder, but it might be possible to have overlays that share the same base to be folders under a common root folder.
I was holding off that reworks as I was hoping with https://github.com/tektoncd/plumbing/issues/262 to start generating those automatically.